components/expenses/MarkExpenseAsUnpaidBtn.js

```diff
@@ -54,7 +54,11 @@ const MarkExpenseAsUnpaidBtn = ({ id, markExpenseAsUnpaid, refetch }) => {
           </StyledButton>
         </Fragment>
       ) : (
-        <StyledButton onClick={() => setState({ ...state, showProcessorFeeConfirmation: true })} mt={2}>
+        <StyledButton
+          buttonStyle="dark"
```

Comment:
Same here, we should keep the default state for this one unless there's a good rational for using a dark style